Cod3rsGrowth.Web/Controllers/EmpresasController.cs

```diff
@@ -38,11 +38,11 @@ public IActionResult Criar([FromBody] Empresa empresaCriar)
         return CreatedAtAction(nameof(ObterPorId), new { id = empresaCriar.Id }, empresaCriar);
```

Comment:
quando vc utilizar um modelo vc vai fazer assim, ou seja definir e buscar o modelo fica centralizado em uma funcao so, cria uma funcao para cada modelo seja quantas for

em todas as contlollers
_modeloEndereco: function (modelo) {
    const nomeModelo = "endereco";
    return this.modelo(nomeModelo, modelo);
},
        
        
coloca na controller base
modelo: function (nome, modelo) {
  let view = this.getView();
  if (modelo) view.setModel(modelo, nome);
  return view.getModel(nome);
},